lessons/misc/quantum-computing/Grover's Algorthim with 2 Qubits.ipynb

```diff
@@ -0,0 +1,396 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simulating Grover's Search Algorithm with 2 Qubits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define the zero and one vectors\n",
+    "Define the initial state $\\psi$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[1]\n",
+      " [0]\n",
+      " [0]\n",
+      " [0]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "zero = np.matrix([[1],[0]]);\n",
+    "one = np.matrix([[0],[1]]);\n",
```

Comment:
Semi-colons not really necessary in the middle of a cell here. Also, please add space after comma everywhere.